x-pack/plugins/cases/server/client/cases/validators.ts

```diff
@@ -133,3 +137,49 @@ export const validateRequiredCustomFields = ({
     );
   }
 };
+
+export const validateSearchCasesCustomFields = ({
```

Comment:
Should we validate here that the request doesn't include duplicated custom field keys?

Is this acceptable `{ "customFields": { "key_1": [null, true], "key_1": [false] } }`?

Reply:
good point, makes sense to add check for duplicate keys.

Reply:
Turns out Object cannot have duplicate keys, 
I passed duplicate keys via postman. It just took the last value of that key. 🤷‍♀️